SourceX/miniwin/misc_io.cpp

```diff
@@ -137,7 +137,11 @@ WINBOOL CloseHandle(HANDLE hObject)
 	std::unique_ptr<memfile> ufile(file); // ensure that delete file is
 	                                      // called on returning
 	files.erase(file);
+#ifndef SWITCH

```

Comment:
Are these `ifdefs` still necessary?

Reply:
Yes, why would they not be? Nothing has changed.

Reply:
They could have been put there as either a hacky workaround, or just as a shotgun method to trying to resolve an unrelated issue. Quite a few things in the original Switch port were done for that reason and turned out to be unessery.